fury/actors/peak.py

```diff
@@ -0,0 +1,343 @@
+from fury.colormap import boys2rgb, colormap_lookup_table, orient2rgb
+from fury.shaders import attribute_to_actor, load, shader_to_actor
+from fury.utils import apply_affine, numpy_to_vtk_colors, numpy_to_vtk_points
+from vtk.util import numpy_support
+
+
+import numpy as np
+import vtk
+
+
+class PeakActor(vtk.vtkActor):
+    def __init__(self, directions, indices, values=None, affine=None,
+                 colors=None, lookup_colormap=None, linewidth=1):
+        if affine is not None:
+            w_pos = apply_affine(affine, np.asarray(indices).T)
+
+        valid_dirs = directions[indices]
+
+        points_array = []
+        centers_array = []
+        diffs_array = []
```

Comment:
why is it a list? you convert it to `np.array` afterwards so maybe you could create directly the array

Reply:
Changed. Added empty list and filled accordingly.